lib/exception_notifier/email_notifier.rb

```diff
@@ -49,11 +95,19 @@ def background_exception_notification(exception, options={}, default_options={})
 
           private
 
+          # Remove any entries from the 'env' var that are not in the 'whitelisted_env_var' list
+          def whitelist_env(env)
+            env.select do |key, val|
+              #FUTURE(willjr): Why wouldn't you just use `===` instead of testing if is a RegExp?
+              whitelisted_env_vars.any? {|allowed| (allowed.is_a? Regexp) ? key =~ allowed : key == allowed }
```

Comment:
@kuahyeow:  I thought this was true, so dug through the doc's and Ruby code:
- When `allowed` is a Regexp, === is like ((a =~ b) ? true : false)
- When `allowed` is a String, === is like (a == b.to_str)
- When `allowed` is a Symbol, === is (a == b)

Therefore, should we simplify the test there to:
`whitelisted_env_vars.any? {|allowed|  allowed === key }` ?